docs: updates to whats new 7.1 (#26410)
grafana/grafana: docs/sources/guides/whats-new-in-v7-1.md

- [**Query history search**]({{< relref "#query-history-search" >}})
- [**Unification of Explore modes**]({{< relref "#explore-modes-unified" >}})
- [**Elasticsearch- link to another data source from Explore**]({{< relref "#internal-links-for-elasticsearch" >}})
- [**Merge on time transform for the new table panel**]({{< relref "#transformations" >}})
- [**Stat panel text mode**]({{< relref "#stat-panel-text-mode" >}})
- [**Time range picker update**]({{< relref "#time-range-picker-update" >}})
- [**Provisioning of apps**]({{< relref "#provisioning-of-apps" >}})
- [**Azure Monitor Datasource**]({{< relref "#azure-monitor-datasource" >}})
- [**Deep linking for Google Cloud Monitoring (formerly named Google Stackdriver) datasource**]({{< relref "#deep-linking-for-google-cloud-monitoring-formerly-named-google-stackdriver-datasource" >}})
- [**Grafana Enterprise features**]({{< relref "#grafana-enterprise-features" >}})
  - [**Secret management with HashiCorp Vault**]({{< relref "#support-for-hashicorp-vault" >}})
  - [**Monthly schedules in reports**]({{< relref "#support-for-monthly-schedules-in-reports" >}})

> ## Influx Datasource

Reviewer: ```suggestion
## Influx data source
```

Support for Flux and Influx v2 has been added. The InfluxData blog post, [How to Build Grafana Dashboards with InfluxDB, Flux and InfluxQL](https://www.influxdata.com/blog/how-grafana-dashboard-influxdb-flux-influxql/) explains the changes in depth.

## Query history search

In Grafana v 7.1 we are introducing search functionality in Query history. You can search across queries and your comments. It is especially useful in combination with a time filter and data source filter. Read more about [Query history here]({{<relref "../features/explore/index.md#query-history" >}}).

{{< docs-imagebox img="/img/docs/v71/query_history_search.gif" max-width="800px" caption="Query history search" >}}

> ## Explore modes unified

Reviewer: Explore needs to be updated to reflect this change.

Author: @oddlittlebird not sure what you mean?

Reviewer 2: Hmm yeah I don't think we update the general explore docs after this. @kaydelaney can you take a look what needs to be updated there?

Author: Oh, do you mean the docs for Explore are not updated yet?

Reviewer: Yes, and then linked from this document. I thought I saw this doc edit go through. Maybe it hasn't been merged yet?

{{< docs-imagebox img="/img/docs/v71/stat-panel-text-modes.png" max-width="1025px" caption="Stat panel" >}}

## Provisioning of apps

Grafana v7.1 adds support for provisioning of app plugins. This allows app plugins to be configured and enabled/disabled using configuration files. Read more about provisioning of app plugins [here]({{<relref "../administration/provisioning.md#plugins" >}}).

> ## Azure Monitor Datasource

Reviewer: ```suggestion
## Azure Monitor data source
```

Support for multiple dimensions has been added to all services in the Azure Monitor datasource. This means you can now group by more than one dimension with time series queries. With the Kusto based services, Log Analytics and Application Insights Analytics, you can also select multiple metrics as well as multiple dimensions.

> Additionally, the “Raw Edit” mode for Application Insights Analytics has been replaced with a new service in the drop down for the datasource and is called “Insights Analytics”. The new query editor behaves in the same way as Log Analytics.

Reviewer: ```suggestion
Additionally, the Raw Edit mode for Application Insights Analytics has been replaced with a new service in the drop down for the data source and is called Insights Analytics. The new query editor behaves in the same way as Log Analytics.
```

> ## Deep linking for Google Cloud Monitoring (formerly named Google Stackdriver) datasource

Reviewer: ```suggestion
## Deep linking for Google Cloud Monitoring (formerly named Google Stackdriver) data source
```

You can now use HashiCorp Vault to get secrets for configuration and provisioning of Grafana Enterprise. Learn more about this feature [here]({{<relref "../enterprise/vault.md">}}).

### Support for monthly schedules in reports

> With Grafana Enterprise 7.1 you can configure reports to be generated on a [monthly schedule]({{<relref "../enterprise/reporting.md#scheduling">}}).

Reviewer: ```suggestion
With Grafana Enterprise 7.1, you can generate reports on a [monthly schedule]({{<relref "../enterprise/reporting.md#scheduling">}}).
```
